pkg/loadbalancer/loadbalancer.go

```diff
@@ -315,6 +315,36 @@ func IsValidStateTransition(old, new BackendState) bool {
 	return true
 }
 
+func GetBackendState(state string) (BackendState, error) {
```

Comment:
nit: this maybe a little nicer implemented on the `BackendState` type itself.

Reply:
It would look like 

var t BackendState
t.FromString("active") 

If you wanted to fill in the var's mem address with the int representing the incoming state.